config/routes.rb

```diff
@@ -656,7 +656,9 @@
     get '/' => 'start_page#show'
 
     get '/accessibility', to: 'content#accessibility'
-    get '/privacy-policy', to: 'content#privacy_policy', as: :privacy_policy
```

Comment:
could we add a redirect for this to privacy?

Reply:
Yep, added